lib/async.js

```diff
@@ -15,12 +15,23 @@ function findAllConfigs(options) {
         });
 
         return Promise.all(realFiles.map(function(file) {
-            return fsHelpers.read(file).then(function(content) {
-                var parsedConfig = nodeEval('(' + content + ')');
```

Comment:
looks like `node-eval` works just fine with commonJS as well so these changes are not necessary